src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksTaxRuleDependencyServiceImplTest.java

```diff
@@ -96,5 +98,23 @@ void testBuildDateFromString() {
         
         assertEquals(expectedDate, actualDate);
     }
+    
+    @Test
+    void testpopulateOwernshipCodeLLC_C_Corp() {
+        PaymentWorksVendor pmwVendor = new PaymentWorksVendor();
+        pmwVendor.setRequestingCompanyTaxClassificationCode(PaymentWorksConstants.LLC_TAXED_AS_C_CORPORATION_TAX_CLASSIFICATION_INDICATOR);
+        VendorHeader vendorHeader = new VendorHeader();
+        taxRuleService.populateOwernshipCode(pmwVendor, TaxRule.NOT_INDIVIDUAL_US, vendorHeader);
```

Comment:
I would suggest creating another user story to fix the typos in the service method's name.